Dockerfile

```diff
@@ -4,7 +4,7 @@ ENV APP_DIR /app
 
 RUN apt-get update && \
     apt-get install -y --no-install-recommends \
-                    gcc g++ make git libffi-dev libssl-dev libc6-dev
+                    gcc g++ make git libffi-dev libssl-dev libc6-dev jq==1.5
```

Comment:
Why do we need `jq` now? I mean it's useful - but not obviously used here?

Reply:
Same here.